tests/src/main.rs

```
@@ -13,13 +12,11 @@ fn main() {
     println!("Hello, world!");
 
     let mut f = MyFlags::new();
-    let nested = Nested::new().a().b(Test::SomeRandomName);
+    let nested = Nested::new().a(true).b(Test::SomeRandomName);
```

Clippy (1.95.0)

[warning] unused variable

(warning)